text-ui-test/ACTUAL.txt

```diff
@@ -0,0 +1,17 @@
+    ____________________________________________________________
+     Hello! I'm Duke
+     What can I do for you?
+    ____________________________________________________________
```

Comment:
Would be good to update the test case to be able to test all features of the program